zap/src/main/java/org/zaproxy/zap/control/ExtensionFactory.java

```diff
@@ -325,6 +326,13 @@ private static void loadMessages(Extension ext) {
         }
     }
 
+    private static boolean isSame(ResourceBundle rb1, ResourceBundle rb2) {
```

Comment:
Rename to `isSameResourceBundle` for clarity?

Reply:
Or maybe that's clear from the type of the params.

Reply:
Renamed, better be clear.